mingw-support/xmllite.def

```diff
@@ -0,0 +1,13 @@
+;
```

Comment:
These are very simple def files. I'm not sure what value it provides to have them generated by a tool and what licensing issues may occur as a result. Can we remove the comments?